spec/index.bs

```diff
@@ -835,7 +835,7 @@ interface GPUDevice : GPUObjectBase {
     GPUComputePipeline createComputePipeline(GPUComputePipelineDescriptor descriptor);
     GPURenderPipeline createRenderPipeline(GPURenderPipelineDescriptor descriptor);
 
-    GPUCommandEncoder createCommandEncoder(GPUCommandEncoderDescriptor descriptor);
+    GPUCommandEncoder createCommandEncoder(GPUCommandEncoderDescriptor? descriptor);
```

Comment:
`?` means nullable type https://heycam.github.io/webidl/#idl-nullable-type.

```suggestion
    GPUCommandEncoder createCommandEncoder(optional GPUCommandEncoderDescriptor descriptor);
```

Reply:
Thank you!